content/docs/sidebar.json

```diff
@@ -785,6 +785,10 @@
             "slug": "make_report",
             "label": "make_report()"
           },
+          {
+            "slug": "make_stage_template",
+            "label": "make_stage_template()"
```

Comment:
Q: do we actually need this as a public API though?

Reply:
We don't need it. I was thinking it helps explain things, and it is just a way to be more explicit, like `make_dvcyaml()` and `make_report()`. Maybe it's better to discuss it in https://github.com/iterative/dvc.org/pull/4777? Any reason you don't want it to be public?

Reply:
no strong reason. I guess I would keep it open if we expect people to use it directly, if it's only as a need to explain things - I would keep it simple and remove. Not a blocker, just an observation.

Reply:
Like those other methods, I wouldn't expect people to use it directly most of the time. On the other hand, in some circumstances it might not get written, and people might be frustrated about how to generate it if it's not exposed publicly. Definitely not critical to make it public, that's just my reasoning.